Unix/server/server.c

```diff
@@ -894,6 +895,13 @@ static void GetConfigFileOptions()
             if (SetPathFromNickname(key, value) != 0)
                 err(ZT("SetPathFromNickname() failed"));
         }
+        else if (strcasecmp(key, "NtlmCredsFile") == 0)
+        {
+            if (value)
+            {
+                s_opts.ntlmCredFile = PAL_Strdup(value);
```

Comment:
It looks like you fixed the documentation. That resolves this issue, I think as documented, this would not have worked.